scripts/street_sign_recognizer.py

```diff
@@ -3,29 +3,70 @@
 """ This is a script that walks through some of the basics of working with images
     with opencv in ROS. """
 
-import rospy
+import rospy, os
 from sensor_msgs.msg import Image
 from cv_bridge import CvBridge
 import cv2
 import numpy as np
+import template_matcher as tm
 
 class StreetSignRecognizer(object):
     """ This robot should recognize street signs """
 
 
     def __init__(self):
         """ Initialize the street sign reocgnizer """
-        rospy.init_node('street_sign_recognizer')
-        self.cv_image = None                        # the latest image from the camera
-        self.bridge = CvBridge()                    # used to convert ROS messages to OpenCV
+
+        # Initialize template matcher 
+        dir_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Yay for a clean way to do file paths.  This seems robust to running the node from any directory on the file system.